README.md

```diff
@@ -77,7 +77,7 @@ If you use our model in your own work, please cite the following paper:
 ```bibtex
 @article{wagner2022dawn,
   title={Dawn of the transformer era in speech emotion recognition: closing the valence gap},
-  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn and Burkhart, Felix},
+  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Burkhardt, Felix and Eyben, Florian and Schuller, Bj{\"o}rn W.},
```

Comment:
This is not the order currently shown in the paper.

Reply:
I re-submitted a new version of the paper on arxiv, which changes the author list to:

![image](https://user-images.githubusercontent.com/10383417/158560927-26061a79-cc2d-416a-b7f5-d7d5c2bd3dd4.png)

I changed it accordingly (including the missing W.)

Reply:
New version us up: https://arxiv.org/abs/2203.07378

Reply:
Thanks, updated version is online now.